--- a/config.php
+++ b/config.php
@@ -176,5 +176,10 @@

 /*============================== End of Configuration =================================*/

+require_once(__DIR__.'/../../../../modules/contrib/ckeditor/includes/filemanager.config.php');
+$config['authentication'] = 'CheckAuthentication';
+$config['backends'][0]['root'] = $baseDir;
+$config['backends'][0]['baseUrl'] = $baseUrl;
+
 // Config must be returned - do not change it.
 return $config;
